blog/2021/01/arrow.md

```diff
@@ -0,0 +1,32 @@
+@def title = "Apache Arrow Support in Julia"
+@def authors = "Jacob Quinn"
+@def published = "26 January 2021"
+@def rss_pubdate = Date(2021, 1, 26)
+@def rss = """Apache Arrow Support in Julia"""
+
+We're excited to announce official support for the Julia language in the [Apache Arrow]() project. Since the [1.0 release](https://arrow.apache.org/blog/2020/07/24/1.0.0-release/) of the arrow format last July (2020), there have been calls for support in Julia. While Julia supported the original [feather v1](https://github.com/JuliaData/Feather.jl) format very early on, support for the "upgraded" arrow format was delayed as the format matured.
+
+[Initial support was merged](https://github.com/apache/arrow/pull/8547) into the Apache Arrow master branch last November, and iteration on the package API and bugfixes have continued at a fast pace since then. While most language implementations all live in the [apache/arrow monorepo](https://github.com/apache/arrow), initial work on the Julia package was done in the [JuliaData/Arrow.jl repo](https://github.com/JuliaData/Arrow.jl). The code is still being actively developed in the Arrow.jl repository, with snapshots merged "upstream" for major releases. Users can choose to install the official Apache Arrow Julia code, or follow the latest releases of the same codebase in the Arrow.jl repository.
+
+The arrow format is a modern-day accumulation of a number of format innovations from the last 30 years. It stores actual data in binary representation, which allows avoiding costly serialization/deserializtaion computing when transfering data between processes or to disk and back. It stores important metadata about the data using the [google flatbuffers format](https://google.github.io/flatbuffers/), including column names, types, and other important metadata.
+
+The Julia implementation of the arrow format is done in pure Julia. It provides Julia `AbstractVector` objects for referencing data that conforms to the Arrow standard.  This allows users to seamlessly interface Arrow formatted data with a great deal of existing Julia code. It also implements the [Tables.jl interface](https://tables.juliadata.org/stable/) for the `Arrow.Stream` and `Arrow.Table` objects, which allows seamless integration with a [growing number](https://github.com/JuliaData/Tables.jl/blob/master/INTEGRATIONS.md) of integrations, including: [DataFrames.jl](https://github.com/JuliaData/DataFrames.jl), [JSONTables.jl](https://github.com/JuliaData/JSONTables.jl), [JuliaDB.jl](https://github.com/JuliaData/JuliaDB.jl), [SQLite.jl](https://github.com/JuliaDatabases/SQLite.jl), [MySQL.jl](https://github.com/JuliaDatabases/MySQL.jl), [JDBC.jl](https://github.com/JuliaDatabases/JDBC.jl), [ODBC.jl](https://github.com/JuliaDatabases/ODBC.jl), [XLSX.jl](https://github.com/felipenoris/XLSX.jl), and many more.
```

Comment:
> This allows users to seamlessly interface Arrow formatted data with a great deal of existing Julia code

I think it might be helpful to expand on this a bit more, since I think it's really a killer feature of the Julia implementation. E.g. things like https://arrow.apache.org/docs/cpp/compute.html are unnecessary since standard Julia code works (and should be as performant). I think that's something that readers who are new to Julia might not pick up on.

(This is something @jrevels pointed out to me! Although I was coming from the Julia side and didn't realize other languages needed such things aha)

Reply:
Great suggestion; I've added a bit more there w/ an example.